keepassxc-browser/options/options.html

```diff
@@ -2,439 +2,1034 @@
+                  for="showLoginNotifications"
+                ></label>
+              </div>
+
+              <div class="form-group mt-3">
+                <label for="blinkTimeout" data-i18n="optionsLabelBlinkTime"></label>
+                <div class="input-group col-lg-3 col-md-4 col-12">
+                  <input
+                    type="number"
+                    id="blinkTimeout"
+                    placeholder="7500"
+                    value="7500"
+                    min="-1"
+                    class="form-control"
+                    aria-describedby="optionsLabelBlinkTimeAppend"
+                  />
+                  <div class="input-group-append">
+                    <span class="input-group-text" id="optionsLabelBlinkTimeAppend">ms</span>
+                  </div>
+                </div>
+                <small class="form-text text-muted">
+                  <span data-i18n="optionsBlinkTimeHelpText"></span>
+                  <br />
+                  <span data-i18n="optionsDefault" i18n-placeholder="7500"></span>,
+                  <span data-i18n="optionsInfinite" i18n-placeholder="-1"></span>
+                </small>
+                <button class="btn btn-sm btn-primary mt-2" id="blinkTimeoutButton" type="button">
+                  <i class="fa fa-floppy-o" aria-hidden="true"></i>
+                  <span data-i18n="optionsButtonSave"></span>
+                </button>
+              </div>
+
+              <div class="form-group">
+                <label for="blinkMinTimeout" data-i18n="optionsLabelRedirectOffset"></label>
+                <div class="input-group col-lg-3 col-md-4 col-12">
+                  <input
+                    type="number"
+                    id="blinkMinTimeout"
+                    placeholder="2000"
+                    value="2000"
+                    min="-1"
+                    class="form-control"
+                    aria-describedby="optionsLabelRedirectOffsetAppend"
+                  />
+                  <div class="input-group-append">
+                    <span class="input-group-text" id="optionsLabelRedirectOffsetLabel">ms</span>
+                  </div>
+                </div>
+                <small class="form-text text-muted">
+                  <span data-i18n="optionsRedirectOffsetHelpText"></span>
+                  <br />
+                  <span data-i18n="optionsRedirectOffsetHelpTextSecond"></span>
+                  <br />
+                  <span data-i18n="optionsDefault" i18n-placeholder="-1"></span>,
+                  <span data-i18n="optionsRecommended" i18n-placeholder="2000"></span>
+                </small>
+                <button
+                  class="btn btn-sm btn-primary mt-2"
+                  id="blinkMinTimeoutButton"
+                  type="button"
+                >
+                  <i class="fa fa-floppy-o" aria-hidden="true"></i>
+                  <span data-i18n="optionsButtonSave"></span>
```

Comment:
Warnings are not displayed when the value is invalid.

Reply:
Also, why the buttons are at the bottom, not after the input field? Then for example "Default group for saving new passwords" has the save button next to the input.

Reply:
The buttons are now placed directly behind the inputs.
The "Blink Time" and "Redirect Offset" options look a bit weird with the `ms` text addon. Should it be removed?

![Screenshot_2019-06-02_18-53-17](https://user-images.githubusercontent.com/19861998/58764500-0811fb80-8568-11e9-8e79-61bdf1ed4ce5.png)

Reply:
The "ms" is already explained in the help text.